libs/shared/src/consts/providers/provider.enum.ts

```diff
@@ -1,6 +1,10 @@
 /* eslint-disable @typescript-eslint/naming-convention */
 
 export enum CredentialsKeyEnum {
+  AuthKey = 'authKey',
```

Comment:
Is this key a public key? Is a secret key? Depends on what is its goal we can either use the existing `ApiKey` or `SecretKey` credential keys.

Reply:
No this is a secret key. It will be better if we will use AuthKey. Yes It's working is similar to the ApiKey

Reply:
I am sorry but I do not understand your response.
If `AuthKey` is a secret key that can't be public, please us the Credential key `SecretKey`. If `AuthKey` is an API key that can be public, please use the Credential key `ApiKey`.

Reply:
If this was decided to be replaced with APi Key enum, can you please remove it from here?